src/main/java/io/personium/core/model/jaxb/Acl.java

```diff
@@ -156,7 +156,12 @@ public static Acl fromJson(final String jsonString) {
             //  attr somehow not unmarshalled so manually fix the object
             JSONParser parser = new JSONParser();
             JSONObject j = (JSONObject) parser.parse(jsonString);
-            ret.setRequireSchemaAuthz((String) j.get(KEY_REQUIRE_SCHEMA_AUTHZ));
+            // -- TODO Delete this block after 1.8.x
+            // backward compatibility for json data before 1.7.20
+            if (ret.requireSchemaAuthz == null) {
+                ret.setRequireSchemaAuthz((String) j.get(KEY_LEGACY_REQUIRE_SCHEMA_AUTHZ));
```

Comment:
(*) this is necessary because the requireSchemaAuthz attribute in the ACL will be stored with a new key.

-  New Key:  "@p.requireSchemaAuthz"
-  Old Key:  "@requireSchemaAuthz"